### PR TITLE
tcp MinMaxConnectionDurationS filter addition

### PR DESCRIPTION
**Pull request by Tejesh-SH**

### CODE REVIEW

File: ETWAnalyzer_uTest/EventDump/DumpTcpTests.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        [Fact]
+        public void MinMaxConnectionDurationFilterTest()
+        {
+            DumpTcp tcpDumper = new();
+            
+            var data = CreateTestData();
+         
+            tcpDumper.MinMaxConnectionDurationS = new MinMaxRange<double>(0, 10);
+
+            Assert.True(tcpDumper.MinMaxConnectionDurationFilter(data[0].Connection.TimeStampOpen, data[0].Connection.TimeStampClose));
```
Comment (AloisKraus) [inline ETWAnalyzer_uTest/EventDump/DumpTcpTests.cs:99]: the passed data can be directly added to the method. It would be better to declare the timestamps and connections inside the method with descriptive names. Currently it is hard to see if the MinMaxConnectionDurationFilter works correctly. 

internal bool MinMaxConnectionDurationFilter(DateTimeOffset? connectTime, DateTimeOffset? closeTime) expects only primitive types.

You can therefore do 
DateTimeOffset connect_0 = new DateTimeOffset(2000,1,1,0,0,0,TimeSpan.Zero);
DateTimeOoffset connect_5s = connect_0 + TimeSpan.FromSeconds(5);
DateTimeOfffset connect_10s= connect_0 + TimeSpan.FromSeconds(10); 
DateTimeOfffset connect_11s= connect_0 + TimeSpan.FromSeconds(11); 

Then you can write it like this: 
   Assert.IsTrue( tcpDumper.MinMaxConnectionDurationFilter(connect_0, connect_5s) );
Then it is much more obvious which timestamps are tested. 
Then you do not need CreateTestData

File: ETWAnalyzer_uTest/EventDump/DumpTcpTests.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                {
+                    Connection = tcpConnection2,
+                }
+            };
+
+            return data;
+        }
+
+        [Fact]
+        public void MinMaxConnectionDurationFilterTest()
```
Comment (AloisKraus) [inline ETWAnalyzer_uTest/EventDump/DumpTcpTests.cs:91]: Please create two tests. One with matching values and the other with not matching like
ConnectionDuration_Matching()
ConnectionDurationFilter_NotMatching()

to test the corresponding corner cases.
